src/cpp/session/modules/SessionFind.cpp

```diff
@@ -1431,6 +1432,7 @@ core::Error runGrepOperation(const GrepOptions& grepOptions, const ReplaceOption
    }
    else
    {
+      cmd << "-E"; // use extended-grep (egrep) for Extended Regular Expressions
```

Comment:
Same as above.

Reply:
(You might want to combine with https://github.com/rstudio/rstudio/pull/10362/files?w=1#diff-94062616737b3005a9fa89d60aa0596694818c97563995b50cba4661956d69dbR1448-R1449)